ci/vulcan/build-for-ci.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Build the webring at a particular commit hash and put it in the `/dev/shm/webring-ci` directory.
+# 
+# The working directory is expected to be the highest directory of the git repo (~/webring)
+
+if [ $# -ne 1 ]; then
+  echo Expected one argument for the commit hash
+  exit 1
+fi
+
+HASH=$1
+
+# Set up the testing directory in tmpfs
+
+CI_PATH=/dev/shm/webring-ci
```

Comment:
Why are we recloning every time instead of resetting a long-lived worktree?

Reply:
Technically it wouldn't be taking up storage, and instead taking up RAM.